config/initializers/companies.yml

```diff
@@ -182,11 +182,11 @@
 -
   name:       'honestbee'
   website:    'https://www.honestbee.com'
-  logo_url:   'https://cloud.githubusercontent.com/assets/12945452/10658532/765e7e3c-78ca-11e5-8090-892ec9912f19.png'
-  address:    '2 Alexandra Road, #04-01B, Singapore 159919'
-  hiring_url:
-  email:      'edmund.yee@honestbee.com'
-  updated_on: '2017-01-01'
+  logo_url:   'https://user-images.githubusercontent.com/1129607/33252835-6d8f6c86-d37b-11e7-9642-784c0e792d21.png'
+  address:    '2 Alexandra Road, #04-01B Delta House, Singapore 159919'
+  hiring_url: 'http://careers.honestbee.com/'
```

Comment:
@jlowhy would `http://careers.honestbee.com/departments/engineering/` be a more specific URL?

Reply:
Makes sense. Let's use that instead.